### PR TITLE
add continuous value model op

### PR DESCRIPTION
**Pull request by colourful-tree**

### CODE REVIEW

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -10561,3 +10562,50 @@ def tree_conv(nodes_vector,
...
         pre_activation = out
     return helper.append_activation(pre_activation)
+
+
+def cvm(input, cvm, use_cvm=True):
+    """
+    **cvm layers**
+
+    This layer accepts a tensor named input which is ID after embedded and lod level is 1 ,
+         cvm is a show_click info and returns the cvm output.
```
Comment (panyx0718) [inline python/paddle/fluid/layers/nn.py]: not sure what does "cvm is a show_click info and returns the cvm output" mean. what is `cvm`?

File: python/paddle/fluid/layers/nn.py [hunk 2]
```diff
@@ -10561,3 +10562,50 @@ def tree_conv(nodes_vector,
...
+
+    Args:
+        input (Variable):  a 2-D LodTensor with shape [N x D], where N is the
+                                batch size, D is the embedding dim. 
+                                lod level = 1.
+        cvm (Variable):    a 2-D Tensor with shape [N x 2], where N is the batch size, 2 is show and click.
+        use_cvm  (bool):  use cvm or not. if use cvm, the output dim is the same as input
```
Comment (panyx0718) [inline python/paddle/fluid/layers/nn.py:10925]: do you think it's confusing? how can cvm layer not using cvm?

File: python/paddle/fluid/layers/nn.py (hunk 1 above)
Comment (colourful-tree) [reply python/paddle/fluid/layers/nn.py]: cvm is continuous value moded. now, it only consider show and click value in ctr project.

File: python/paddle/fluid/layers/nn.py (hunk 2 above)
Comment (colourful-tree) [reply python/paddle/fluid/layers/nn.py:10925]: yeah，it's a little confusing. cvm op is a  customized op, which input is a sequence had embedd_with_cvm default, so we need a op named cvm to decided whever use it or not.

**Review by panyx0718 — DISMISSED** [review]

**Comment by shanyi15** [comment]
please preview the comment first, thanks

**Pull request closed without merging**